.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy Preview
+on: pull_request
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: S3
+    env:
+      PUBLIC_URL: http://poject-milou-frontend.s3-website.eu-central-1.amazonaws.com/preview/${{ github.event.number }}
+    steps:
+      - uses: actions/checkout@v1
+      
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+           ${{ runner.os }}-yarn-
+           
+      - name: Set deployment status
+        uses: pbrandone/create-status-action@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          state: pending
+          description: Preparing deploy preview
+          context: Deploy Preview URL
+          
+      - name: Install
+        run: yarn install
+
+      - name: Build
+        run: yarn run build
+
+      - name: Deploy
+        if: success()
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: poject-milou-frontend
```

Comment:
Same typo here. If fixing this is much effort and it's spelled like this everywhere, I'd leave it.